cracking-the-coding-interview/problems/2.linked_lists/7.palindrome.ts

```diff
@@ -0,0 +1,40 @@
+// 7. *Palindrome*: Implement a function to check if a linked list is a palindrome.
+
+
+class LinkedNode {
+  value;
+  next;
+  constructor(value) {
+    this.value = value;
+    this.next = null; // Initialize 'next' as null
+  }
+}
+
+const a = new LinkedNode('a');
+const b = new LinkedNode('b');
+const c = new LinkedNode('c');
+const d = new LinkedNode('c');
+const e = new LinkedNode('b');
+const f = new LinkedNode('a');
+
+a.next = b;
+b.next = c;
+c.next = d;
+d.next = e;
+e.next = f;
+
+const checkPalindrome = (node) => {
+  const arr = []
```

Comment:
Can we solve the problem without using another data structure? 

I think this is a sliding window challenge.